client/app/components/auth/login-dialog.js

```diff
@@ -1,6 +1,22 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import ENV from 'labs-zap-search/config/environment';
 
+const MAINTENANCE_RANGE = ['06/28/2021 19:00', '06/29/2021 19:00'];
```

Comment:
What timezone is this? Should you use UTC timestamps instead? Does this default to UTC?

Reply:
probably. browsers defaults to the user's local timezone